diff -up bsnes_v085-source/bsnes/ui/general/main-window.cpp.systemwide bsnes_v085-source/bsnes/ui/general/main-window.cpp
--- bsnes_v085-source/bsnes/ui/general/main-window.cpp.systemwide	2011-12-11 18:40:56.000000000 +0100
+++ bsnes_v085-source/bsnes/ui/general/main-window.cpp	2012-01-03 19:57:44.770296862 +0100
@@ -332,7 +332,7 @@ void MainWindow::synchronize() {
 }
 
 void MainWindow::setupVideoFilters() {
-  string path = { application->basepath, "filters/" };
+  string path = { "/usr/lib/bsnes/filters/" };
   lstring files = directory::files(path, "*.filter");
   if(files.size() == 0) {
     path = { application->userpath, "filters/" };
@@ -363,7 +363,7 @@ void MainWindow::setupVideoFilters() {
 }
 
 void MainWindow::setupVideoShaders() {
-  string path = { application->basepath, "shaders/" };
+  string path = { "/usr/share/bsnes/shaders/" };
   lstring files = directory::files(path, { "*.", config->video.driver, ".shader" });
   if(files.size() == 0) {
     path = { application->userpath, "shaders/" };
diff -up bsnes_v085-source/bsnes/ui/Makefile.systemwide bsnes_v085-source/bsnes/ui/Makefile
--- bsnes_v085-source/bsnes/ui/Makefile.systemwide	2011-12-30 07:01:55.000000000 +0100
+++ bsnes_v085-source/bsnes/ui/Makefile	2012-01-03 19:59:35.298641852 +0100
@@ -67,11 +67,8 @@ endif
 install:
 ifeq ($(platform),x)
 	install -D -m 755 out/$(name) $(DESTDIR)$(prefix)/bin/$(name)
-	mkdir -p ~/.config/$(name)
 	install -D -m 644 data/$(name).png $(DESTDIR)$(prefix)/share/pixmaps/$(name).png
 	install -D -m 644 data/$(name).desktop $(DESTDIR)$(prefix)/share/applications/$(name).desktop
-	cp data/cheats.xml ~/.config/$(name)/cheats.xml
-	chmod 777 ~/.config/$(name) ~/.config/$(name)/cheats.xml
 endif
 
 uninstall:
diff -up bsnes_v085-source/bsnes/ui/tools/cheat-database.cpp.systemwide bsnes_v085-source/bsnes/ui/tools/cheat-database.cpp
--- bsnes_v085-source/bsnes/ui/tools/cheat-database.cpp.systemwide	2011-12-31 09:05:26.000000000 +0100
+++ bsnes_v085-source/bsnes/ui/tools/cheat-database.cpp	2012-01-03 20:02:23.651120923 +0100
@@ -34,7 +34,7 @@ void CheatDatabase::findCodes() {
   cheatCode.reset();
 
   string data;
-  data.readfile(application->path("cheats.xml"));
+  data.readfile("/usr/share/bsnes/cheats.xml");
   XML::Document document(data);
   for(auto &node : document["database"]) {
     if(node.name != "cartridge") continue;
