diff -up bsnes-0.068/bsnes/libco/libco.c.noppcelfppc64 bsnes-0.068/bsnes/libco/libco.c
--- bsnes-0.068/bsnes/libco/libco.c.noppcelfppc64	2010-08-21 18:59:27.000000000 +0200
+++ bsnes-0.068/bsnes/libco/libco.c	2010-08-24 10:45:19.000000000 +0200
@@ -8,7 +8,7 @@
   #include "x86.c"
 #elif defined(__GNUC__) && defined(__amd64__)
   #include "amd64.c"
-#elif defined(__GNUC__) && defined(__powerpc__) && defined(__ELF__)
+#elif defined(__GNUC__) && defined(__PPC32__) && defined(__ELF__)
   #include "ppc-elf.c"
 #elif defined(__GNUC__)
   #include "sjlj.c"
