diff -up bsnes_v079-source/bsnes/ui/Makefile.nocheats bsnes_v079-source/bsnes/ui/Makefile
--- bsnes_v079-source/bsnes/ui/Makefile.nocheats	2011-03-15 00:58:22.000000000 +0100
+++ bsnes_v079-source/bsnes/ui/Makefile	2011-06-08 00:09:34.125573793 +0200
@@ -96,9 +96,6 @@ ifeq ($(platform),x)
 endif
 	install -D -m 644 data/bsnes.png $(DESTDIR)$(prefix)/share/pixmaps/bsnes.png
 	install -D -m 644 data/bsnes.desktop $(DESTDIR)$(prefix)/share/applications/bsnes.desktop
-	mkdir -p ~/.config/bsnes
-	cp data/cheats.xml ~/.config/bsnes/cheats.xml
-	chmod 777 ~/.config/bsnes ~/.config/bsnes/cheats.xml
 
 uninstall:
 ifeq ($(platform),x)
