diff -up bsnes-0.070/bsnes/Makefile.nogconftool bsnes-0.070/bsnes/Makefile
--- bsnes-0.070/bsnes/Makefile.nogconftool	2010-09-29 20:36:49.000000000 +0200
+++ bsnes-0.070/bsnes/Makefile	2010-09-29 20:36:59.000000000 +0200
@@ -64,7 +64,6 @@ ifeq ($(platform),x)
 	install -D -m 755 out/bsnes $(DESTDIR)$(prefix)/bin/bsnes
 	install -D -m 644 ui-qt/data/bsnes.png $(DESTDIR)$(prefix)/share/pixmaps/bsnes.png
 	install -D -m 644 ui-qt/data/bsnes.desktop $(DESTDIR)$(prefix)/share/applications/bsnes.desktop
-	gconftool-2 --type bool --set /desktop/gnome/interface/menus_have_icons true
 endif
 
 uninstall:
