diff -up bsnes-0.054/src/lib/libco/libco.c.noppcelf bsnes-0.054/src/lib/libco/libco.c
--- bsnes-0.054/src/lib/libco/libco.c.noppcelf	2009-10-18 09:26:09.000000000 +0200
+++ bsnes-0.054/src/lib/libco/libco.c	2009-10-21 16:01:54.000000000 +0200
@@ -8,8 +8,6 @@
   #include "x86.c"
 #elif defined(__GNUC__) && defined(__amd64__)
   #include "amd64.c"
-#elif defined(__GNUC__) && defined(__powerpc__) && defined(__ELF__)
-  #include "ppc-elf.c"
 #elif defined(__GNUC__)
   #include "sjlj.c"
 #elif defined(_MSC_VER) && defined(_M_IX86)
