diff -up bsnes_v086-source/bsnes/nall/string.hpp.gcc47 bsnes_v086-source/bsnes/nall/string.hpp
--- bsnes_v086-source/bsnes/nall/string.hpp.gcc47	2012-02-13 05:51:19.000000000 +0100
+++ bsnes_v086-source/bsnes/nall/string.hpp	2012-02-17 19:00:18.721016544 +0100
@@ -25,8 +25,8 @@
 #include <nall/string/base.hpp>
 #include <nall/string/bml.hpp>
 #include <nall/string/bsv.hpp>
-#include <nall/string/core.hpp>
 #include <nall/string/cast.hpp>
+#include <nall/string/core.hpp>
 #include <nall/string/compare.hpp>
 #include <nall/string/convert.hpp>
 #include <nall/string/cstring.hpp>
