diff -up bsnes-0.068/bsnes/libco/ppc-elf.c.newppcelf bsnes-0.068/bsnes/libco/ppc-elf.c
--- bsnes-0.068/bsnes/libco/ppc-elf.c.newppcelf	2010-08-21 18:59:27.000000000 +0200
+++ bsnes-0.068/bsnes/libco/ppc-elf.c	2010-08-24 10:45:19.000000000 +0200
@@ -66,9 +66,15 @@ typedef void * cothread_t;
 /*
  * co_active_context is either in a global offset table (if we are
  * compiling -fPIC or -fPIE) or has an absolute position.
+ *
+ * __attribute__((__used__)) is necessary because newer versions of
+ * gcc might replace co_active_context with an inline constant. This
+ * optimization would cause a linker error. This is because gcc cannot
+ * detect that our __asm__ uses co_active_context.
  */
 static void *co_main_stack_pointer;
-static cothread_t co_active_context = &co_main_stack_pointer;
+static cothread_t co_active_context __attribute__((__used__)) =
+  &co_main_stack_pointer;
 
 extern cothread_t co_active() {
   return co_active_context;
