diff -up bsnes_v079-source/bsnes/snes/chip/icd2/interface/interface.cpp.gcc46 bsnes_v079-source/bsnes/snes/chip/icd2/interface/interface.cpp
--- bsnes_v079-source/bsnes/snes/chip/icd2/interface/interface.cpp.gcc46	2011-05-08 13:14:20.000000000 +0200
+++ bsnes_v079-source/bsnes/snes/chip/icd2/interface/interface.cpp	2011-06-21 00:02:22.972853738 +0200
@@ -101,7 +101,7 @@ bool ICD2::input_poll(unsigned id) {
     case 3: data = ~r6007; break;
   }
 
-  switch(id) {
+  switch((GameBoy::Input)id) {
     case GameBoy::Input::Start:  return data & 0x80;
     case GameBoy::Input::Select: return data & 0x40;
     case GameBoy::Input::B:      return data & 0x20;
