diff -up bsnes-0.072/bsnes/Makefile.nocheats bsnes-0.072/bsnes/Makefile
--- bsnes-0.072/bsnes/Makefile.nocheats	2010-11-17 22:46:25.000000000 +0100
+++ bsnes-0.072/bsnes/Makefile	2010-11-17 22:49:08.000000000 +0100
@@ -64,9 +64,6 @@ ifeq ($(platform),x)
 	install -D -m 755 out/bsnes $(DESTDIR)$(prefix)/bin/bsnes
 	install -D -m 644 data/bsnes.png $(DESTDIR)$(prefix)/share/pixmaps/bsnes.png
 	install -D -m 644 data/bsnes.desktop $(DESTDIR)$(prefix)/share/applications/bsnes.desktop
-	test -d ~/.bsnes || mkdir ~/.bsnes
-	cp data/cheats.xml ~/.bsnes/cheats.xml
-	chmod 777 ~/.bsnes ~/.bsnes/cheats.xml
 endif
 
 uninstall:
