diff -up bsnes_v079-source/bsnes/ui/Makefile.gtk bsnes_v079-source/bsnes/ui/Makefile
--- bsnes_v079-source/bsnes/ui/Makefile.gtk	2011-06-22 21:42:22.509169945 +0200
+++ bsnes_v079-source/bsnes/ui/Makefile	2011-06-22 21:42:51.361077057 +0200
@@ -7,11 +7,11 @@ ui_objects += $(if $(call streq,$(platfo
 
 # platform
 ifeq ($(platform),x)
-# phoenix_compile = $(call compile,-DPHOENIX_GTK `pkg-config --cflags gtk+-2.0`)
-# link += `pkg-config --libs gtk+-2.0`
+  phoenix_compile = $(call compile,-DPHOENIX_GTK `pkg-config --cflags gtk+-2.0`)
+  link += `pkg-config --libs gtk+-2.0`
 
-  phoenix_compile = $(call compile,-DPHOENIX_QT `pkg-config --cflags QtCore QtGui`)
-  link += `pkg-config --libs QtCore QtGui`
+# phoenix_compile = $(call compile,-DPHOENIX_QT `pkg-config --cflags QtCore QtGui`)
+# link += `pkg-config --libs QtCore QtGui`
 
   ruby := video.glx video.xv video.sdl
   ruby += audio.alsa audio.openal audio.oss audio.pulseaudio audio.pulseaudiosimple audio.ao
