diff -up bsnes-0.060/src/Makefile.dso bsnes-0.060/src/Makefile
--- bsnes-0.060/src/Makefile.dso	2010-02-21 10:39:54.000000000 +0100
+++ bsnes-0.060/src/Makefile	2010-02-21 10:40:21.000000000 +0100
@@ -25,7 +25,7 @@ link  :=
 ################
 
 ifeq ($(platform),x)
-  link += -s
+  link += -s -ldl -lX11
 
   ruby := video.glx video.xv video.qtraster video.sdl
   ruby += audio.alsa audio.openal audio.oss audio.pulseaudio audio.pulseaudiosimple audio.ao
