--- src/Makefile.system-zlib	2009-05-10 19:20:21.000000000 +0200
+++ src/Makefile	2009-05-10 19:20:21.000000000 +0200
@@ -71,6 +71,7 @@
 ifeq ($(enable_gzip),true)
   objects += adler32 compress crc32 deflate gzio inffast inflate inftrees ioapi trees unzip zip zutil
   flags += -DGZIP_SUPPORT
+  link += $(shell pkg-config --libs minizip)
 endif
 
 ifeq ($(enable_jma),true)
--- src/lib/libreader/gzreader.hpp.system-zlib	2009-05-02 18:18:10.000000000 +0200
+++ src/lib/libreader/gzreader.hpp	2009-05-10 19:21:27.000000000 +0200
@@ -1,4 +1,4 @@
-#include <zlib/zlib.h>
+#include <zlib.h>
 
 class GZReader : public Reader {
 private:
--- src/lib/libreader/zipreader.hpp.system-zlib	2009-05-02 18:18:02.000000000 +0200
+++ src/lib/libreader/zipreader.hpp	2009-05-10 19:22:37.000000000 +0200
@@ -1,4 +1,4 @@
-#include <zlib/unzip.h>
+#include <minizip/unzip.h>
 
 #define ZIP_MAX_FILE_NAME PATH_MAX
 
