diff -up bsnes-0.059/src/Makefile.strip bsnes-0.059/src/Makefile
--- bsnes-0.059/src/Makefile.strip	2010-01-07 03:55:02.000000000 +0100
+++ bsnes-0.059/src/Makefile	2010-01-07 20:21:26.000000000 +0100
@@ -23,7 +23,7 @@ link  :=
 ################
 
 ifeq ($(platform),x)
-  link += -s
+# link += -s
 
   ruby := video.glx video.xv video.qtraster video.sdl
   ruby += audio.alsa audio.openal audio.oss audio.pulseaudio audio.pulseaudiosimple audio.ao
