diff -up bsnes-0.064/src/libco/libco.c.noppcelfppc64 bsnes-0.064/src/libco/libco.c
--- bsnes-0.064/src/libco/libco.c.noppcelfppc64	2010-04-12 06:29:18.000000000 +0200
+++ bsnes-0.064/src/libco/libco.c	2010-04-17 14:37:11.000000000 +0200
@@ -8,7 +8,7 @@
   #include "x86.c"
 #elif defined(__GNUC__) && defined(__amd64__)
   #include "amd64.c"
-#elif defined(__GNUC__) && defined(__powerpc__) && defined(__ELF__)
+#elif defined(__GNUC__) && defined(__PPC32__) && defined(__ELF__)
   #include "ppc-elf.c"
 #elif defined(__GNUC__)
   #include "sjlj.c"
